include/ccf/service/tables/gov.h

```diff
@@ -31,10 +31,14 @@ namespace ccf
     DECLARE_JSON_OPTIONAL_FIELDS(Failure, trace);
     using VoteFailures = std::unordered_map<ccf::MemberId, Failure>;
 
+    /// Proposal metadata stored in the KV
     struct ProposalInfo
     {
+      /// Member ID of the proposer
       ccf::MemberId proposer_id;
+      /// Proposal state
       ccf::ProposalState state;
+      /// Ballots (scripts) submitted for the proposal
```

Comment:
imho those descriptions do not add a lot given the names and types. "Javascript modules exporting a single 'vote' function" would be a more useful description for ballot for example.

Reply:
I'd taken these from the now-deleted type, but really just wanted the `struct` level docstring to remember which of these types is used in which situations. Have expanded the field descriptions on this type, since it's part of the documented governance KV schema, and removed the redundant descriptions on the `ProposalInfoSummary` fields.